### ISSUE TITLE
[Bug]: Change in SponsorBlock API breaks integration

### ISSUE THREAD

**Issue by bbilly1** [tubearchivist/tubearchivist#397]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read through the [wiki](https://github.com/tubearchivist/tubearchivist/wiki) and the [readme](https://github.com/tubearchivist/tubearchivist#installing-and-updating), particularly the [common errors](https://github.com/tubearchivist/tubearchivist#common-errors) section.

### Operating System

linux

## Describe the bug
Sponsorblock changed their API, breaking our integration.

## Steps To Reproduce
Activate integration on settings page and download any video that has segments registered in sponsorblock.

## Expected behavior
Clean output and store relevant fields.

### Relevant log output

```shell
tubearchivist  | [2022-12-31 08:06:59,973: WARNING/ForkPoolWorker-4] xxxxxxxxxxx: get sponsorblock timestamps
tubearchivist  | [2022-12-31 08:07:00,453: ERROR/ForkPoolWorker-4] Task download_pending[925b2a8e-9cd6-493a-bdcf-e7f0aaf20a4f] raised unexpected: KeyError('userID')
tubearchivist  | Traceback (most recent call last):
tubearchivist  |   File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
tubearchivist  |     R = retval = fun(*args, **kwargs)
tubearchivist  |   File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
tubearchivist  |     return self.run(*args, **kwargs)
tubearchivist  |   File "/app/home/tasks.py", line 90, in download_pending
tubearchivist  |     downloader.run_queue()
tubearchivist  |   File "/app/home/src/download/yt_dlp_handler.py", line 210, in run_queue
tubearchivist  |     vid_dict = index_new_video(
tubearchivist  |   File "/app/home/src/index/video.py", line 405, in index_new_video
tubearchivist  |     video.build_json()
tubearchivist  |   File "/app/home/src/index/video.py", line 158, in build_json
tubearchivist  |     self._get_sponsorblock()
tubearchivist  |   File "/app/home/src/index/video.py", line 359, in _get_sponsorblock
tubearchivist  |     sponsorblock = SponsorBlock().get_timestamps(self.youtube_id)
tubearchivist  |   File "/app/home/src/index/video.py", line 70, in get_timestamps
tubearchivist  |     sponsor_dict = self._get_sponsor_dict(all_segments)
tubearchivist  |   File "/app/home/src/index/video.py", line 81, in _get_sponsor_dict
tubearchivist  |     del segment["userID"]
tubearchivist  | KeyError: 'userID'
```

**Comment by bbilly1** [comment]
You have a few options:
- Deactivate the integration for now until next regular release: recommended
- Recreate the fix as in the commit linked above on your own installation: If you know what you are doing
- Pull the `:unstable` image: Only if you are feeling adventurous.

**Comment by davralin** [comment]
Is there any reason why there can't just be an updated release with this builtin?

**Comment by bbilly1** [comment]
We are in the midst of some major refactoring. If you are in a hurry, you can help with testing the unstable builds, join us on Discord. 

In any case sponsorblock is a very nice but third party integration, not core functionality. Use any of the outlined workarounds above to get things working again.

**Comment by bbilly1** [comment]
Fix is now published in v0.3.1, please update.

**Comment by davralin** [comment]
Thanks a bunch, I realised I'm really dependent on this software, so I become a sponsor.
It's not much, but it's also so little I'll probably forget about it!

Thanks for a great piece of software!
